wiki/Help_centre/Upgrading_to_lazer/vi.md

```diff
@@ -0,0 +1,288 @@
+---
+tags:
+  - client trò chơi
+---
+
+# Nâng cấp lên lazer
+
+osu!(lazer) là bản cập nhật quan trọng tiếp theo của trò chơi. Nó là kết quả của nhiều năm làm việc một cách bí mật để làm lại trò chơi một cách tỉ mỉ.
+
+Mục tiêu cuối cùng là phiên bản này sẽ thay thế hoàn toàn phiên bản ổn định của trò chơi, nhưng vẫn còn một vài việc phải hoàn thành trước khi nó khả thi. Cuối cùng, **người chơi** sẽ quyết định khi nào việc này sẽ xảy ra và chúng tôi sẽ tiếp tục hỗ trợ bản phát hành trước đó cho đến khi người dùng đã chuyển qua.
+
+"lazer" là một mật danh và sau dùng sẽ trở thành bản phát hành căn bản của trò chơi. Phần còn lại của bài viết này sẽ gọi osu!(lazer) là "lazer" và osu!(ổn định) là "ổn định" cho đơn giản.
+
+## So sánh các tính năng
+
+Sau đây là bảng so sánh **trạng thái hiện tại** của lazer so với ổn định. Lưu ý rằng đây là một mục tiêu không cố định — mục tiêu cuối cùng là thực hiện tất cả những tính năng mà người dùng quan tâm.
+
+### Khả năng tương thích và hiệu suất
+
+| Tính năng | ổn định | lazer |
+| :-- | :-- | :-- |
+| Windows 8.0 trở xuống | ![Yes][true] | ![No][false] |
+| macOS / Linux | ![Partial][partial][^wine] | ![Yes][true] |
+| DirectX / Vulkan | ![Partial][partial][^compatibility-mode] | ![No][false][^coming-soon] |
+| Hỗ trợ điện thoại | ![No][false] | ![Yes][true] |
+| Kiến trúc đa luồng | ![No][false] | ![Yes][true] |
+| Video sử dụng tăng tốc phần cứng | ![No][false] | ![Yes][true] |
+| Chỉnh sửa giao diện người dùng | ![No][false] | ![Yes][true] |
+| Các bộ quy tắc tuỳ chỉnh (các chế độ chơi) | ![No][false] | ![Partial][partial][^dll] |
+| Kho lưu trữ tệp không trùng lặp | ![No][false] | ![Yes][true][^share-files] |
+| Điều chỉnh vùng bảng vẽ | ![No][false] | ![Yes][true] |
```

Comment:
Missing translation of the word within first brackets [], same for all tables.

Reply:
got it